front/public/cards/hic.json

```diff
@@ -148,5 +148,6 @@
             "name": "Documentation",
             "link": "https://github.com/stjude/proteinpaint/wiki/Interaction-Tracks:-Hi%E2%80%90C-and-Arc"
         }
-    ]
+    ],
+    "citation_id": 1007
```

Comment:
also cite this in arc track card

Reply:
This is addressed in https://github.com/stjude/proteinpaint/pull/2335/commits/5ced6f857bc73a0e41d9867a0046203f1a3bc157